pvl/abridged/math/numtheory/prime-sieve.cpp

```diff
@@ -1,12 +1,8 @@
-std::bitset<N> is; // #include <bitset>
-int pr[N], primes = 0;
+bitset<N> isc; // #include <bitset>
+vector<int> p;

```

Comment:
`std::`?

Reply:
oh do we `std::`  in abridged?

Reply:
For consistency haha. But, ideally, we shouldn't. I guess it we should start not using it na in abridged from now on

Up to you if you wanna add it; just resolve the thread whenever you want